Gemfile

```diff
@@ -21,25 +21,23 @@ gem 'net-ldap'
 
 # Use puma as the webserver in development
 gem 'puma'
-gem 'rack', '1.6.4' # Remove in rails 5
 gem 'rack-cors', :require => 'rack/cors'
 gem 'grape-middleware-lograge'
 gem "rack-timeout"
 
 gem 'grape-swagger'
 gem 'kaminari'
+gem 'kaminari-grape' #needed for kaminari 1.x
 gem 'grape-kaminari'
```

Comment:
The newest release of grape-kaminari appears to need kaminari-grape in order to work correctly.